src/sage/combinat/plane_partition.py

```diff
@@ -324,12 +324,10 @@ def cells(self) -> list[list[int]]:
             sage: PP.cells()
             [[0, 0, 0], [0, 0, 1], [0, 0, 2], [0, 1, 0], [1, 0, 0], [1, 0, 1]]
         """
-        L = []
-        for r in range(len(self)):
-            for c in range(len(self[r])):
-                for h in range(self[r][c]):
-                    L.append([r, c, h])
-        return L
+        return [[r, c, h]
```

Comment:
I think we should be returning tuples here, really.  We do so for partitions and tableaux, too.